packages/wonder-blocks-search-field/src/components/__tests__/search-field.test.js

```diff
@@ -321,6 +310,40 @@ describe("SearchField", () => {
         );
 
         // Assert
-        expect(ref.current?.value).toBe("some-value");
+        waitFor(() => {
+            expect(ref.current?.value).toBe("some-value");
+        });
+    });
+
+    test("uses the passed in ID if one is provided", () => {
+        // Arrange
+        render(
+            <SearchField
+                id={"some-random-id"}
+                testId="search-field-test"
+                onChange={() => {}}
+                value=""
+            />,
+        );
+
+        // Act
+        const searchField = screen.getByTestId("search-field-test");
+
+        // Assert
+        expect(searchField).toHaveAttribute("id", "some-random-id-field");
+    });
+
+    test("uses a unique ID if one is not provided", () => {
+        render(
```

Comment:
Nice test! One minor thing... could you please add `// Arrange, Act, Assert` comments on this test?